header.php

```diff
@@ -11,7 +11,7 @@
 		<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
 		<meta name="description" content="<?php bloginfo('description'); ?>">
-
+		<script src="https://kit.fontawesome.com/01463a92e6.js" crossorigin="anonymous"></script>
```

Comment:
I just got my fontawesome cdn working perfectly. We can use this or mine...it doesn't really matter.